Externals/SlippiLib/SlippiGame.cpp

```diff
@@ -6,11 +6,13 @@
 
 namespace Slippi {
 
+  static uint8_t* data;
+
   //**********************************************************************
   //*                         Event Handlers
   //**********************************************************************
   //The read operators will read a value and increment the index so the next read will read in the correct location
-  uint8_t readByte(uint8_t* a, int& idx, uint32_t maxSize, uint8_t defaultValue) {
+  static uint8_t readByte(uint8_t* a, int& idx, uint32_t maxSize, uint8_t defaultValue) {
```

Comment:
a lot of changes with adding `static` to functions or variables that are only used within the current file (which is what declaring functions static is for)